driver-core/src/main/com/mongodb/operation/AggregateExplainOperation.java

```diff
@@ -153,7 +159,7 @@ public BsonDocument execute(final ReadBinding binding) {
             @Override
             public BsonDocument call(final Connection connection) {
                 validateCollation(connection, collation);
-                return executeWrappedCommandProtocol(binding, namespace.getDatabaseName(), getCommand(), connection);
+                return CommandOperationHelper.executeCommand(binding, retryReads, namespace.getDatabaseName(), getCommand(), connection);
```

Comment:
Nit: statically import the method

Reply:
Done. IntelliJ did this when refactoring, so it occurred in a number of files.